jit/v2/JIT_v2.0.0.yaml

```diff
@@ -1159,25 +1162,26 @@ components:
         - portCallID
         - carrierServiceName
         - carrierServiceCode
-      ########################
-      # Transport Call Service
-      ########################
-    TransportCall:
+
+    ########################
+    # Terminal Call Service
+    ########################
+    TerminalCall:
       type: object
-      title: Transport Call
+      title: Terminal Call
       description: |
-        **Transport Call** information. The `transportCallID` uniquely identifies the **Transport Call**. Any subsequent **Port Call Services** linked to the same **Transport Call** MUST use the same `transportCallID`. An optional `transportCallReference` can be added in order to link the **Transport Call** to external systems (e.g OVS).
+        **Terminal Call** information. The `terminalCallID` uniquely identifies the **Terminal Call**. Any subsequent **Port Call Services** linked to the same **Terminal Call** MUST use the same `terminalCallID`. An optional `transportCallReference` can be added in order to link the **Terminal Call** to external systems (e.g OVS).
       properties:
-        transportCallID:
+        terminalCallID:
           type: string
           format: uuid
           description: |
-            Universal unique identifier for the **Transport Call**. The `transportCallID` is created by the provider. The `transportCallID` **MUST** only be created once per **Transport Call** - which is the first time a **Port Call Service** is also created. To be used in all communication regarding the **Transport Call**.
+            Universal unique identifier for the **Terminal Call**. The `terminalCallID` is created by the provider. The `terminalCallID` **MUST** only be created once per **Terminal Call** - which is the first time a **Port Call Service** is also created. To be used in all communication regarding the **Terminal Call**.
           example: 0342254a-5927-4856-b9c9-aa12e7c00563
         transportCallReference:
           type: string
           description: |
-            The unique reference for a **Transport Call**. It’s the vessel operator’s responsibility to provide the **Transport Call Reference**, other parties are obliged to pick it up and use it. It can take the form of **Port Call References** as defined in OVS Definitions Document, or alternatively a reference as defined by the vessel operator.
+            The unique reference for a **Terminal Call**. It’s the vessel operator’s responsibility to provide the **Terminal Call Reference**, other parties are obliged to pick it up and use it. It can take the form of **Port Call References** as defined in OVS Definitions Document, or alternatively a reference as defined by the vessel operator.
           maxLength: 100
           example: SR11111X-9321483-2107W-NLAMS-ACT-1-1
```

Comment:
I would rather use a reference as typically used by carriers, the format defined by DCSA is not widely used and should probably change in the future. We allow for carrier created references simpler to create, i.e.: 15063401

Reply:
Not sure what you are asking? I will undo the change and insert **Transport Call** again..

Reply:
I was talking about the example for the transport call reference.